crates/ln-dlc-node/src/ln/coordinator_event_handler.rs

```diff
@@ -422,10 +428,15 @@ where
         );
     }
 
-    // TODO(holzeis): check liquidity option here
-    let channel_value = expected_outbound_amount_msat / 1000 * crate::LIQUIDITY_MULTIPLIER;
+    let expected_outbound_amount =
+        Decimal::from(expected_outbound_amount_msat) / Decimal::from(1000);
+    let trade_up_to = Decimal::from(liquidity_request.trade_up_to);
+    let coordinator_leverage =
+        Decimal::try_from(liquidity_request.coordinator_leverage).expect("to fit into decimal");
 
     let fake_scid = FakeScid::new(requested_next_hop_scid);
+    let channel_value = expected_outbound_amount + (trade_up_to / coordinator_leverage);
```

Comment:
I guess this assumes that the coordinator wants to have the same leverage for all channels. Might be a bit limiting, but we can start with this.

Reply:
not for all channels, but for all positions of that channel. I think it needs to be that way otherwise we can not hold up to the trade up to value selected in the liquidity option. 

maybe we could increase it.